packages/transport/src/types/messages.ts

```diff
@@ -554,8 +554,8 @@ export type FirmwareErase = {
 
 // FirmwareRequest
 export type FirmwareRequest = {
-    offset?: number;
-    length?: number;
+    offset: number;
```

Comment:
I'd suggest moving these changes to another PR. we shouldn't need to update submodules -> messages.json at all. do we?

Reply:
Not really, but I did not want to update it manually so I changed the patch for `EthereumGetAddress` to have the right type ` encoded_network?: ArrayBuffer;` and this is the result of the automatic script. 
But you are right, it is not part of this PR. So I am including only the required bit.